README.md

```diff
@@ -105,11 +105,11 @@ If there are multiple destination registries, the above calculation is repeated
 for each destination registry. The promoter also prints warnings about images in
 `M` that cannot be found in the source registry (call it `S`):
```

Comment:
Does "it" refer to the images in the source registry?

Reply:
> Does "it" refer to the images in the source registry?

Yes.

At the same time, the logic around using Set difference for calculating what needs to be copied over was dropped in https://github.com/kubernetes-sigs/k8s-container-image-promoter/pull/86/commits/b062e7a33898604917394986036d26c15c476562, so the language in the README using these concepts should be removed.